src/app/modules/submission/encrypt-submission/encrypt-submission.utils.ts

```diff
@@ -237,3 +240,26 @@ export const createEncryptedSubmissionDto = (
     version: submissionData.version,
   }
 }
+
+export const getPaymentAmount = (
```

Comment:
shall we add a UT for the new method?

Reply:
Added! 793abffb1933385ed6cbde30a5a5e92c518922d1